zipline/errors.py

```diff
@@ -433,10 +433,20 @@ class TermInputsNotSpecified(ZiplineError):
     msg = "{termname} requires inputs, but no inputs list was passed."
 
 
+class TermOutputsNotSpecified(ZiplineError):
```

Comment:
This should be `TermOutputsEmpty`, not `NotSpecified`.  `NotSpecified` means they weren't passed at all.